charts/validator/Chart.yaml

```diff
@@ -22,11 +22,11 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.4.0-pre.1
+version: 0.4.0-pre.2
 
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "0.58.0"
+appVersion: "0.58"
```

Comment:
no patch?

Reply:
thhis is appVersion. so it includes all 58.* since the image is pinned in values yaml with digest